jsapp/scss/components/_kobo.map.scss

```diff
@@ -400,6 +400,27 @@ body .leaflet-control-layers,
   }
 }
 
+.map-settings__querylimit {
+  label {
+    display: block;
+    margin-bottom: 10px;
+
+    input {
+      margin-right: 8px;
+    }
+  }
+  .change-limit-warning {
+    color: red;
+  }
+  .change-limit-slider {
+    width: 100%;
+  }
+  .change-limit-slider-value {
+    display: block;
+    text-align: center;
+  }
```

Comment:
The slider `input` and `output` could really be on the same line - just make the slider `width: 75%`, add some left margin to value and set `dispay: inline-block; vertical-align: middle;` to both.